File: app/src/main/java/com/sagar/watchnext/network/models/people/Persons.java

package com.sagar.watchnext.network.models.people;

import java.util.List;

import com.google.gson.annotations.SerializedName;

public class Persons {

    @SerializedName("page")
    private int page;

    @SerializedName("results")
    private List<Person> persons;

    @SerializedName("total_results")
    private int totalResults;

    @SerializedName("total_pages")
    private int totalPages;

    public int getPage() {
        return page;
    }


    public List<Person> getPersons() {
        return persons;
    }


    public int getTotalResults() {
        return totalResults;
    }


    public int getTotalPages() {
        return totalPages;
    }

}

File: app/src/main/java/com/sagar/watchnext/network/models/people/Persons.kt
package com.sagar.watchnext.network.models.people

import com.google.gson.annotations.SerializedName

class Persons {

    @SerializedName("page")
    val page: Int = 0

    @SerializedName("results")
    val persons: List<Person>? = null

    @SerializedName("total_results")
    val totalResults: Int = 0

    @SerializedName("total_pages")
    val totalPages: Int = 0

}
